tests/typescript_decorators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -49,8 +49,7 @@ export class TabCompletionController {}
   selector: "angular-component"
 })
 class AngularComponent {
-  @Input()
-  myInput: string;
+  @Input() myInput: string;
```

Comment:
This looks like the changes from #5188, rebasing issue?

Reply:
Oh probably